syfoalerts-sbs.yaml

```diff
@@ -11,10 +11,10 @@ spec:
       #prependText: '<!here> | '
   alerts:
     - alert: syfo-sbs-app-nede
-      expr: up{app=~"sykefravaerarbeidsgiver",job="kubernetes-pods"} == 0
+      expr: kube_deployment_status_replicas_available{deployment=~"sykefravaerarbeidsgiver"} == 0
```

Comment:
Kanskje vi burde ha lagt inn sykefravaer også (altså den vanlige frontenden for dittsykefravær)?

Reply:
Jepp, legger til sykefravaer